delegate/delegate/ViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ViewController.swift
+//  delegate
+//
+//  Created by Risako Kato on 2022/08/03.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    let fruitsList: [String] = ["りんご🍎", "ぶどう🍇", "みかん🍊", "いちご🍓", "さくらんぼ🍒"]
+
+    @IBOutlet weak var fruitsCollectionView: UICollectionView!
+    @IBOutlet weak var collectionViewFlowLayout: UICollectionViewFlowLayout!
```

Comment:
@risako070310 weakつけてるの丁寧でいいね！業務のコードでは `@IBOutlet private weak var`などとweakを記述します。

なんだけど、toyboxではわかりやすさ重視なのでLife is Tech!の他の教科書と揃えてweakを削除していただけるととても助かりーみんです！

```suggestion
    @IBOutlet var fruitsCollectionView: UICollectionView!
    @IBOutlet var collectionViewFlowLayout: UICollectionViewFlowLayout!
```